src/pytest/test_safemode.py

```diff
@@ -1,8 +0,0 @@
-# -*- coding: utf-8 -*-
-from test import SearchTestCase
-import os
-
-class SafemodeTestCase(SearchTestCase):
-    # TODO: Implement a proper API in rmtest and expose this correctly
```

Comment:
so no tests for safe mode?

Reply:
No, I'll need to figure out how to do this